README.md

```diff
@@ -75,8 +75,8 @@ matching dependencies:
 ```
 cd $GOPATH/src/github.com/decred/dcrd
 git pull
-glide install
-go install $(glide nv)
+dep ensure -update
```

Comment:
This would cause the lock file to be updated and is not correct for simply ensuring the dependencies match the project's lock file.  It should just be `dep ensure`.